### PR TITLE
document manual resource installation

### PR DESCRIPTION
**Pull request by wwwtyro**
See https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/180

### CODE REVIEW

File: fragments/k8s/cdk/README.md [hunk 1]
```diff
@@ -495,11 +495,17 @@ documentation for more information.
...
 
  - Deployment locally on LXD will require the use of conjure-up to tune
    settings on the host's LXD installation to support Docker and other
-   componentry.
+   components.
+
+ - If resources fail to download during initial deployment for any reason, you
+   will need to download and install them manually. For example, if
+   kubernetes-master is missing its resources, download them
+   [here](https://jujucharms.com/u/containers/kubernetes-master/) and install
```
Comment (chuckbutler) [inline fragments/k8s/cdk/README.md]: its not clear to an end user that the resources on the right hand side are what they are looking for. I wonder if there's more verbiage here we can use to guide them to look at that specific area of the store display.

Should we also link to the individual charms somewhere here in the readme so they don't have to guess/search where they need to go for resources?

**Comment by wwwtyro** [comment]
@mbruzek, yeah, I'll update all the fragments with their locations if they have resources in the container namespace.

File: fragments/k8s/cdk/README.md [hunk 2]
```diff
@@ -495,11 +495,25 @@ documentation for more information.
...
+   section of the sidebar [here](https://jujucharms.com/u/containers/kubernetes-master/)
+   and install them by running
+   `juju attach kubernetes-master kubernetes=/path/to/kubernetes-master.tar.gz`.
+
+   You can find the resources for CDK charms here:
+
+   - [kubernetes-master](https://jujucharms.com/u/containers/kubernetes-master/)
+   - [kubernetes-worker](https://jujucharms.com/u/containers/kubernetes-worker/)
+   - [easyrsa](https://jujucharms.com/u/containers/easyrsa/)
+   - [etcd](https://jujucharms.com/u/containers/etcd/)
```
Comment (mbruzek) [inline fragments/k8s/cdk/README.md]: I think you are missing the link to the flannel resource here too.

File: fragments/k8s/core/README.md [hunk 3]
```diff
@@ -404,11 +402,26 @@ to determine fault tolerance.
...
+   section of the sidebar [here](https://jujucharms.com/u/containers/kubernetes-master/)
+   and install them by running
+   `juju attach kubernetes-master kubernetes=/path/to/kubernetes-master.tar.gz`.
+
+   You can find the resources for CDK Core charms here:
+
+   - [kubernetes-master](https://jujucharms.com/u/containers/kubernetes-master/)
+   - [kubernetes-worker](https://jujucharms.com/u/containers/kubernetes-worker/)
+   - [easyrsa](https://jujucharms.com/u/containers/easyrsa/)
+   - [etcd](https://jujucharms.com/u/containers/etcd/)
```
Comment (mbruzek) [inline fragments/k8s/core/README.md]: Also mention where the flannel resource is here.

**Comment by mbruzek** [comment]
+1 to the latest edition.

**Pull request merged**